packages/flutter_tools/lib/src/commands/analyze.dart

```diff
@@ -131,6 +139,44 @@ class AnalyzeCommand extends FlutterCommand {
           'all available build variants',
       hide: !verboseHelp,
     );
+
+    if (verboseHelp) {
+      argParser.addSeparator('Usage: flutter analyze --ios [arguments]');
+    }
+
+    argParser.addFlag('list-build-options',
+      help: 'Print out a list of available build options for the '
+          'iOS Xcode sub-project.',
+      hide: !verboseHelp,
+    );
+
+    argParser.addFlag('output-universal-link-settings',
+      negatable: false,
+      help: 'Output a JSON with iOS Xcode universal link settings into a file. '
+          'The "--configuration", "--scheme", and "--target" must also be set.',
+      hide: !verboseHelp,
+    );
+
+    argParser.addOption('configuration',
+      help: 'Sets the iOS build configuration to be analyzed.',
+      valueHelp: 'use "flutter analyze --ios --list-build-options" to get '
```

Comment:
So this long string is messing up verbose help formatting (the android --build-variant help also does this, I must not have checked for that PR). Can you make `valueHelp` shorter, and then move these instructions to the `help` field? Something like:

`valueHelp: '<configuration>',`

Reply:
![Screenshot from 2023-09-06 13-30-08](https://github.com/flutter/flutter/assets/7856010/a32d92cd-fe90-44c1-bb7a-c0c872f8da33)

Reply:
oops I will do